vignettes/traitstrap-workflow.Rmd

```diff
@@ -190,14 +234,18 @@ sum_boot_moment
 
 ## Parametric bootstrapping
 
-Traitstrap also has the option to run a **parametric bootstrapping**.
+Traitstrap also offers the option to run a **parametric bootstrapping**.
 
 The **trait_fit_distributions** function fits parametric distributions for each species-by-trait combination at the finest scale of the user-supplied hierarchy. 
 This function takes as input:
-1) an object of class imputed traits (as produced by the function trait_impute), and 
-2) the type of distribution to be fitted.
-Either a single distribution type can be used for all traits, or traits can be assigned specific distributions types by supplying the function with a named list of traits.
-Currently supported distribution types are normal, log-normal, and beta.
+
+* an object of class imputed traits (as produced by the function trait_impute), and 
+* the type of distribution to be fitted.
+
+Either a single distribution type can be used for all traits, or traits can be assigned specific distributions types by supplying the function with a named list of traits (e.g. list(height = "normal", mass = "lognormal")).
+
+Currently the function supports normal, log-normal, and beta distributions.
```

Comment:
may be worth pointing out either here or in the help file that the beta dist needs values in the range 0-1